internal/pkg/cli/env_init.go

```diff
@@ -468,10 +467,16 @@ https://aws.amazon.com/premiumsupport/knowledge-center/ecs-pull-container-api-er
 		return fmt.Errorf("VPC %s has no DNS support enabled", o.importVPC.ID)
 	}
 	if o.importVPC.PublicSubnetIDs == nil {
-		publicSubnets, err := o.selVPC.PublicSubnets(envInitPublicSubnetsSelectPrompt, "", o.importVPC.ID)
+		input := selector.SubnetsInput{
```

Comment:
nit: can we remove the variable `o.selVPC.Subnets({...})`

Reply:
Sorry-- I'm not really sure what you mean.... 😬

Reply:
🤣 sorry that wasn't clear! 

```go
o.selVPC.Subnets(selector.SubnetsInput{
	Msg:      envInitPublicSubnetsSelectPrompt,
	VPCID:    o.importVPC.ID,
	IsPublic: true,
})
```